ATM_Selenium_Training/src/test/java/com/epam/atm/selenium/NewAppTest.java

```diff
@@ -0,0 +1,72 @@
+package com.epam.atm.selenium;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.regex.Pattern;
+
+public class NewAppTest extends BaseTest {
+
+    @Test
+    public void newApplication() {
+        int x = (int) (Math.random() * 1000);
+        driver.get("https://network-international--qa.cs83.my.salesforce.com/");
+        System.out.println("Open application (url) for testing");
+        Assert.assertEquals(driver.getTitle(), "Login | Salesforce", "Incorrect page (login page) is not opened");
+        System.out.println("Check that correct Login page is opened");
+        WebElement username = driver.findElementByXPath("//input[@class='input r4 wide mb16 mt8 username']");
+        username.sendKeys("yauhen_valodzin@epam.com.qa");
```

Comment:
there are cases when it could be some data in field so better first clear and then input

Reply:
as clear and input are two steps u can create a separate method for imputing data in the field
public inputData(WebElement el) {
el.clear()
el.sendKeys()
}